### PR TITLE
Added linear residual output

### PR DESCRIPTION
**Pull request by NasserFlexCompute**

### CODE REVIEW

File: flow360/component/flow360_params/flow360_output.py [hunk 1]
```diff
@@ -780,8 +780,11 @@ class LegacyOutputFormat(pd.BaseModel, metaclass=ABCMeta):
     primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
     q_criterion: Optional[bool] = pd.Field(alias="qcriterion")
     residual_navier_stokes: Optional[bool] = pd.Field(alias="residualNavierStokes")
+    linear_residual_navier_stokes: Optional[bool] = pd.Field(alias="linearResidualNavierStokes")
```
Comment (benflexcompute) [inline flow360/component/flow360_params/flow360_output.py:783]: Why do we need to add these for legacy output?

Comment (NasserFlexCompute) [reply flow360/component/flow360_params/flow360_output.py:783]: This is mostly for internal use and thought would be good to be able to add them in old jsons as well. Should we remove them?

Comment (benflexcompute) [reply flow360/component/flow360_params/flow360_output.py:783]: These old JSONs are still using boolean to indicate what fields to output? Like 
`"primitiveVars": True`?

Comment (benflexcompute) [reply flow360/component/flow360_params/flow360_output.py:783]: If they already use "outputFields" then we do not need these

**Pull request closed without merging**